index.js

```diff
@@ -132,69 +124,56 @@ function onExecute(env) {
     process.env.UNDERTAKER_SETTLE = 'true';
   }
 
+  if (optsErr) {
+    log.error(msgs.error.failToParseCliOpts, optsErr.message);
+    makeHelp(parser).showHelp(console.error);
+    exit(1);
+  }
   if (env.config.flags.help) {
-    parser.showHelp(console.log);
+    makeHelp(parser).showHelp(console.log);
     exit(0);
   }
 
   // Anything that needs to print outside of the logging mechanism should use console.log
   if (env.config.flags.version) {
-    console.log('CLI version:', cliVersion);
-    console.log('Local version:', env.modulePackage.version || 'Unknown');
+    var gulpVersion = env.modulePackage.version || 'Unknown';
+    console.log(format(theme, msgs.info.version, cliVersion, gulpVersion));
```

Comment:
I don't think we want to allow this to be configured. We ask for this specific information in our bug reports.

Anywhere we use `console.log` instead of the logger we don't need to support configuration.

Reply:
Understood. I think that's reasonable.